src/NServiceBus.Core.Tests/Sagas/ComponentTests/When_completing_a_saga_with_no_defined_correlation_property.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_completing_a_saga_with_no_defined_correlation_property : SagaPersisterTests<SagaWithoutCorrelationProperty, SagaWithoutCorrelationPropertyData>
```

Comment:
I wouldn't say that, don't we want to test names to be as accurate as possible? (In this case to not give people reading the code the impression that we support running with no correlation property)